static/app/components/events/eventAttachments.tsx

```diff
@@ -48,7 +48,7 @@ class EventAttachments extends React.Component<Props, State> {
       case 'text/json':
       case 'text/x-json':
       case 'application/json':
-        if (attachment.name === 'rrweb.json') {
+        if (attachment.name === 'rrweb.json' || attachment.name.startsWith('rrweb-')) {
```

Comment:
Bad news for anyone who randomly prefixed their totally unrelated attachment with `rrweb-`.

Nothing to do here, just acknowledging this very remote possibility.